### PR TITLE
New landing page, tweetie gone? 

### PR DESCRIPTION
**Pull request by sveranicola**

### CODE REVIEW

**Review by Stavai — APPROVED** [review]
bird is the word

File: src/components/LandingPage.jsx [hunk 1]
```diff
@@ -48,10 +48,10 @@ var LandingPage = ({ isGoogleSignedIn, setIsGoogleSignedIn }) => {
 
   return (
     <div className='landing-page'>
-      <div className='header'></div>
+      {/* <div className='header'></div> */}
```
Comment (GriffGeorgiadis) [inline src/components/LandingPage.jsx:51]: pls fix

File: src/landingStyle.css [hunk 2]
```diff
@@ -26,13 +29,11 @@ span.icon {
...
 }
 span.buttonText {
+  background: white;
   display: inline-block;
   vertical-align: middle;
-  /* padding-left: 42px;
-  padding-right: 42px; */
   font-size: 14px;
   font-weight:100;
-  /* Use the Roboto font that is loaded in the <head> */
```
Comment (GriffGeorgiadis) [inline src/landingStyle.css:35]: pls fix

File: src/landingStyle.css [hunk 3]
```diff
@@ -26,13 +29,11 @@ span.icon {
   height: 42px;
 }
 span.buttonText {
+  background: white;
   display: inline-block;
   vertical-align: middle;
-  /* padding-left: 42px;
```
Comment (GriffGeorgiadis) [inline src/landingStyle.css:31]: pls fix

File: src/components/LandingPage.jsx [hunk 4]
```diff
@@ -79,7 +79,7 @@ var LandingPage = ({ isGoogleSignedIn, setIsGoogleSignedIn }) => {
           Please login with google
         </div>
       </div>
-      <div className='footer'></div>
+      {/* <div className='footer'></div> */}
```
Comment (GriffGeorgiadis) [inline src/components/LandingPage.jsx:82]: pls fix

**Review by GriffGeorgiadis — CHANGES_REQUESTED** [review]
pls fix

**Review by GriffGeorgiadis — APPROVED** [review]
LGTM

**Pull request merged**